algo-builder-tester/assets/contracts/Bitcoin-approval.teal

```diff
@@ -0,0 +1,231 @@
+#pragma version 5
+gtxn 0 ApplicationID
```

Comment:
Do you directly write teal instead of pyteal here? looks like naming below `main_l24` etc. is like generated name. If it's generated from a pyteal, the pyteal source need to commited too

Reply:
NO. this is generated by python module "run"

Reply:
ah i see! it's generated from trader contract py <- commented in wrong place

Reply:
It's like [`trade_contract.py`, `asset_config.py`] -> `contract_generator.py` -> `exporter.py`
`A -> B`: B imports A